mlir/test/Conversion/ConvertToAIR/condense_memref_ops_to_air_memcpy.mlir

```diff
@@ -0,0 +1,92 @@
+//===- condense_memref_ops_to_air_memcpy.mlir ------------------*- MLIR -*-===//
+//
+// Copyright (C) 2024, Advanced Micro Devices, Inc. All rights reserved.
+// SPDX-License-Identifier: MIT
+//
+//===----------------------------------------------------------------------===//
+
+// RUN: air-opt %s -air-copy-to-dma -canonicalize -cse | FileCheck %s
+
```

Comment:
Would it be possible to split this into smaller tests (check the canonicalization of a single dma_memcpy at a time) ?